src/utils/nodes/MemberExpression.ts

```diff
@@ -40,13 +40,23 @@ export class MemberExpressionClass extends NodeClass {
 				type: 'push',
 				value:
 					this.args
-						?.map((arg) =>
-							nodeFactory({
-								node: arg,
-								context: this.context,
-								params: this.params,
-							}).serialize()
-						)
+						?.map((arg) => {
+							if (arg.type === 'Identifier') {
+								const variableValue = this.context.variables[arg.name];
+								if (!variableValue) return '';
```

Comment:
Currently, when writing
```
let a;
console.log(a);
```
a blank line is output in the Console. What kind of behavior would be expected?

Reply:
Hey @nm723 thank you for the question.
We want to simulate browser behaviour, therefore `undefined` seems like an appropriate output in such case.

Reply:
Thank you for the advice. I changed the output to `undefined`.